### DISCUSSION TITLE
Should be possible partially drag window off page

### DISCUSSION THREAD

**Discussion by alaindesilets** [nextapps-de/winbox#93] [Ideas]
At the moment when you drag a winbox, it will stop dragging as soon as one of the box's edges hits an edge of the page. 

This is not standard behavior for a window. In a desktop environment, it's possible to drag a window partially off-screen. The window lets itself be dragged as long as there is a bit of the titlebar that is visible. 

I think this should be the case for winboxes also, because it gives the user more flexibility for seeing content that may be partially obstructed by the window. This is particularly useful in a small real-estate situation.

**Comment by ts-thomas** [comment]
This is now supported in v0.2.7 by the new option "overflow" during initialization
